moved/src/state_actor.rs

```diff
@@ -94,14 +94,14 @@ pub struct StateActor<S, P, H, R, G, L1G, L2G, B, Q, M, SQ, T, TQ, N, RR, RQ> {
     execution_payloads: HashMap<B256, PayloadResponse>,
     pending_payload: Option<(PayloadId, PayloadResponse)>,
     mem_pool: HashMap<B256, (ExtendedTxEnvelope, L1GasFeeInput)>,
-    state: S,
+    pub state: S,
```

Comment:
Should we add setters instead of making them public? Since others aren't public either